src/Home.tsx

```diff
@@ -106,42 +96,22 @@ const Home = ({
 		}
 
 		setLoading(true);
-		api('POST', `/chain`, {
-			"id":"1",
-			"jsonrpc":"2.0",
-			"method":"getstorage",
-			"params":[
-				environment.netmapContract,
-				environment.epochLine,
-			],
-		}).then((res: any) => {
-			if (res['result']) {
-				const epoch_b64: any = atob(res['result']);
-				const epoch_bytes: any = base64ToArrayBuffer(epoch_b64);
-				const dv2: any = new DataView(epoch_bytes.buffer);
-				const epoch: string | number = dv2.getUint32(0, true);
-
-				files.forEach((file: File) => {
-					const formData: any = new FormData();
-					formData.append('file', file);
-					onUploadFile(formData, file.name, epoch, lifetimeData);
-				});
-			} else {
-				setLoading(false);
-				onModal('failed', 'Something went wrong, try again');
-			}
-		}).catch(() => {
-			setLoading(false);
-			onModal('failed', 'Something went wrong, try again');
+		files.forEach((file: File) => {
+			const formData: any = new FormData();
+			formData.append('file', file);
+			onUploadFile(formData, file.name);
 		});
     e.preventDefault();
 	};
 
-	const onUploadFile = (formData: any, filename: string | null, epoch: string | number, lifetime: string | number) => {
-		document.cookie = `Bearer=${user.XBearer}; path=/gate/upload; expires=${new Date(Date.now() + 10 * 1000).toUTCString()}`;
-		api('POST', '/gate/upload/', formData, {
-			'X-Attribute-NEOFS-Expiration-Epoch': String(Number(epoch) + Number(lifetime)),
-			'X-Attribute-email': user.XAttributeEmail,
+	const onUploadFile = (formData: any, filename: string | null) => {
+		api('POST', `/objects/${environment.dataCid}`, formData, {
+			'X-Attributes': JSON.stringify({
+				'filename': filename,
```

Comment:
This better be FileName, now we have these attributes defined exactly, case-sensitive.